sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClient.java

```diff
@@ -67,7 +68,7 @@
      * @param disableBufferCopy Determines whether deep cloning of response buffers should be disabled.
      */
     NettyAsyncHttpClient(reactor.netty.http.client.HttpClient nettyClient, boolean disableBufferCopy,
-                         long readTimeout, long writeTimeout, long responseTimeout) {
+        long readTimeout, long writeTimeout, long responseTimeout) {
```

Comment:
fwiw, I prefer the previous style whenever I see you making these changes....the indentation makes it clearer that these are method parameters and not the first line of the method.

Reply:
> fwiw, I prefer the previous style whenever I see you making these changes....the indentation makes it clearer that these are method parameters and not the first line of the method.

Sounds good with me, we should look into having a standardized `.editorconfig` or IntelliJ code style configuration that should be a requirement when developing SDKs. Additionally, do we want to update Checkstyle rules to enforce this pattern?

Reply:
I've got plans underway to standardise this.